src/javascript/binary/websocket_pages/trade/common.js

```diff
@@ -914,7 +914,7 @@ function chartFrameSource() {
 }
 
 function setChartSource() {
-  document.getElementById('chart_frame').src = 'https://webtrader.binary.com?affiliates=true&instrument=' + document.getElementById('underlying').value + '&timePeriod=' + document.getElementById('time_period').value + '&gtm=true';
+  document.getElementById('chart_frame').src = 'https://webtrader.binary.com?affiliates=true&instrument=' + document.getElementById('underlying').value + '&timePeriod=' + document.getElementById('time_period').value + '&gtm=true&lang=' + page.language().toLowerCase();
```

Comment:
@negar-binary Is it possible that `page.language()` could be `undefined` or `null`? 
Do we have to do `(page.language() || 'en').toLowerCase()`?